Mathlib/Algebra/Homology/ShortComplex/LeftHomology.lean

```diff
@@ -1021,6 +1021,29 @@ noncomputable def leftHomologyIsoCokernelLift [S.HasLeftHomology] [HasKernel S.g
     S.leftHomology ≅ cokernel (kernel.lift S.g S.f S.zero) :=
   (LeftHomologyData.ofHasKernelOfHasCokernel S).leftHomologyIso
 
+/- The following lemmas and instance gives a sufficient condition for a morphism
```

Comment:
You could make this
```suggestion
/-! The following lemmas and instance gives a sufficient condition for a morphism
```
so that it shows up in the online docs.